crates/vault-registry/src/lib.rs

```diff
@@ -716,7 +801,11 @@ impl<T: Config> Pallet<T> {
                 );
                 Self::slash_backing_collateral(account, amount)?;
             }
-            CurrencySource::Griefing(_) | CurrencySource::ReservedBalance(_) | CurrencySource::LiquidationVault => {
+            CurrencySource::Griefing(_) => {
+                amount.unlock(&from.account_id())?;
+            }
+            CurrencySource::ReservedBalance(_) | CurrencySource::LiquidationVault => {
+                Self::decrease_total_backing_collateral(amount)?;
```

Comment:
I renamed this to LiquidatedCollateral because this is the only thing we use it for. I kept the decrease, and also added increase when transferring to it